### PR TITLE
Updated list of CC with colors according to the result status.

### LINKED ISSUE #337: Improve header of new HTML report: Mark passed, failed and skipped conformance classes
Header of list of conformance classes should be updated to:
```Core Conformance Classes (Pass = Green; Fail = Red; Skip = Grey):```

In the following list all conformance classes should be marked by a colour resulting from test result. Colour should be encoded in HTML:
Pass = Green
Fail = Red
Skip = Grey

Example for list (here GeoPackage 1.2 NSG test suite). Colours are not displayed here:

```
Core Conformance Classes (Pass = Green; Fail = Red; Skip = Grey):
  Core
  Features
  Tiles
  Attributes
  Extension Mechanism
  Non-Linear Geometry Types
  RTree Spatial Indexes
  Tiles Encoding WebP
  Metadata
  Schema
  WKT for Coordinate Reference Systems
  Tiled Gridded Elevation Data
  NSG
```

### PR DESCRIPTION
**Pull request by keshav-nangare**
close #337

### CODE REVIEW

File: teamengine-resources/src/main/resources/com/occamlab/te/earl_html_report.xsl [hunk 1]
```diff
@@ -205,6 +196,28 @@
...
+              Core Conformance Classes (Pass = Green; Fail = Red; Skip = Grey):
+              <xsl:for-each select="/rdf:RDF/cite:TestRun/cite:requirements/rdf:Seq/rdf:li/earl:TestRequirement">
+                <xsl:if test="cite:isBasic = 'true'">
+                <xsl:variable name="testsInheritedFailure" select="if (cite:testsInheritedFailure >= 0) then cite:testsInheritedFailure else 0 " />
+                <xsl:variable name="bgcolor" >
+					<xsl:choose>
+						<xsl:when test="cite:testsPassed > 0 and cite:testsFailed = 0 and $testsInheritedFailure = 0">
+							 <xsl:value-of select="string('#B2F0D1')" />
+						</xsl:when>
+						<xsl:when test="cite:testsPassed = 0 and cite:testsFailed > 0 or $testsInheritedFailure > 0">
```
Comment (dstenger) [inline teamengine-resources/src/main/resources/com/occamlab/te/earl_html_report.xsl]: "cite:testsPassed = 0" seems to be not correct as a conformance class is failed even if single tests passed.

**Comment by dstenger** [comment]
@keshav-nangare Can you please review my comments?

**Comment by dstenger** [comment]
@keshav-nangare If you are updating the pull request, can you also improve the formatting by adding an empty line right before
*  ```Test INPUT:```
* ```Result:````
* and ```Core Conformance Classes (Pass = Green; Fail = Red; Skip = Grey):```

**Comment by keshav-nangare** [comment]
> Can you also please add a screenshot to the issue how the current result header looks like?

 
 
Current new HTML header is as below:

image

**Comment by keshav-nangare** [comment]
The report after changes:

image

**Comment by keshav-nangare** [comment]
@dstenger 

Please review the changes.

**Pull request merged**